src/main/java/com/rakovets/course/java/core/practice/enum_types/Season.java

```diff
@@ -0,0 +1,17 @@
+package com.rakovets.course.java.core.practice.enum_types;
+
+public enum Season {
+    WINTER("-15C"),
+    SPRING("+15C"),
+    SUMMER("+25C"),
+    AUTUMN("+12");
+    private final String temperature;
```

Comment:
You need store date in raw type: 

25C -> int temp = 25, TemperatureMeasurement measurement = TemperatureMeasurement.CELSIUS